doc/advanced-topics.rst

```diff
@@ -234,3 +234,18 @@ files can be loaded into Chromium's ``chrome://tracing``. Note that the exact
 format is subject to change between versions.
 
 .. _Catapult trace-viewer: https://github.com/catapult-project/catapult/blob/master/tracing/README.md
+
+No Implicit Transitive Deps (1.7+)
```

Comment:
Other section do not have a version information, I don't think it's necessary to add this.

Reply:
Sure, it does not hurt. But since we're backwards compatible, there's the nice property that the last version includes all the documented features (and trying to use this in an older project will tell you what to change).

Reply:
Personally, I don't mind either way, but if we add this annotation here, we should probably add it in other places as well for consistency.